### DISCUSSION TITLE
Should we hyphenates long names?

### DISCUSSION THREAD

**Discussion by Thomas-Boi** [devicons/devicon#459] [General]
Hello all, 

As you guys know, one of our naming guidelines is that a logo's name is concatenated together, like `amazonwebservices` or `woocommerce`. However, we are thinking of whether to start adding hyphens to our name to make it easier to read. For example, this would make `amazonwebservices` to `amazon-web-services`. We want to discuss with everyone because there are some icons that have very long name, such as `microsoftsqlserver`. This can make it hard to read for some people.

We would like to ask everybody what you guys think about hyphenating names. Do you like the current way we are doing it (no space, no hyphens) or would you like to split it up with hyphens?

The reason why we didn't do this sooner was due to backward compatibility. When the new maintainers start contributing to the repo, there are icons that were done the old way, like the amazon one above. If people are used to search for `amazonwebservices`, it's a bit weird to start searching for hyphenated names.

**Comment by amacado** [comment]
I would suggest to stay with the current schema since it could be confusing to search for icons and changing existing icon names would have the drawback of huge incompatibility. I was thinking of using the `aliases` to allow multiple names for a icon. So both versions would be valid (we might need to have an automated check to not creating duplicated aliases).

**Comment by Thomas-Boi** [comment]
I think that we should stick to the current schema too. Either way, searching for `amazonwebservice` or `amazon-web-service` in our website would yield the Amazon icon after the user enter `amazon` so it's not much of a difference.

About repurposing the `aliases` attribute, I think this is something we should look into. I will open another discussion for the `aliases` topic.
